src/lib/holocene/accordion.svelte

```diff
@@ -68,7 +68,7 @@
         <slot name="summary" />
       </h2>
       <div
-        class="flex flex-row items-center"
+        class="flex flex-row items-center gap-2 pr-2"
```

Comment:
Adds spacing if multiple actions and padding between actions and chevron ⬇️ 

<img width="279" alt="Screenshot 2024-04-22 at 1 01 31 PM" src="https://github.com/temporalio/ui/assets/15069288/98db95e8-38b1-48ae-a388-33681127e3eb">

Reply:
👍